backend/src/main/java/com/google/sps/data/MatchStatus.java

```diff
@@ -18,7 +32,6 @@ public int getValue() {
   }
 
   public static MatchStatus forIntValue(int value) {
-    System.out.println(value);
```

Comment:
nit: In general, it's no harm to keep logs in your program for debugging purpose, so there's usually no need to remove a logging statement. (But I understand we don't use logging module but print to stdout, so removing might make sense here)